src/server/server_state.h

```diff
@@ -203,6 +203,10 @@ class ServerState {  // public struct - to allow initialization.
 
   facade::ConnectionStats connection_stats;
 
+  absl::flat_hash_map<std::string, uint64_t> cmd_calls_map;  // number of calls per command
```

Comment:
Since this is indexing command names, we could do a faster lookup by just storing the pointer to the first character, right? We could have `absl::flat_hash_map<const char*, uint64_t>` and access it by `cid->name().data()`, and we could skip string comparisons.

Reply:
It's a hacky to rely on the implementation detail that all the command names are constant strings, though.

Reply:
yes, i like it. we can add a comment in command_registry about it.

Reply:
Awesome idea!